app/views/wpcc/your_details/_details_form.html.erb

```diff
@@ -0,0 +1,111 @@
+<section class="details" data-dough-component="ConditionalMessaging">
+  <%= form_for(@your_details_form, url: your_details_path) do |f| %>
+    <h2 class="details__heading">1. <%= t('wpcc.details.title') %></h2>
+    <div class="details__row">
+      <div class="details__field">
+        <div class="form__row">
+           <%= f.label :age, t('wpcc.details.age.label'), class: 'form__label-heading' %>
+           <%= f.text_field :age, 'data-dough-age-field': true %>
+        </div>
+      </div>
+
+      <div class="details__helper">
+        <div class="field-help-text">
+          <div class="tooltip__content-container">
+            <p><%= t('wpcc.details.age.tooltip') %></p>
+          </div>
+        </div>
+      </div>
+    </div>
+
```

Comment:
Its worth investigating abstracting the tooltip stuff into its own partial as it appears in this view a few times